src/services/cartItemService.ts

```diff
@@ -77,4 +83,112 @@ export const deleteCartItemService = async (cartId: string) =>{
   }catch(error){
     throw new Error("An error occurred while deleting the cart item");
   }
+};
+
+export const deleteAllCartItemsForUserService = async (userId: string) => {
+  try{
+    const user = await getUserById(userId);
+    if(!user){
+      throw new Error("User not found");
+    }
+    await deleteCartItemsByUserId(userId);
```

Comment:
this doesn't return any thing? i belive it returns the deleted user id you can re-return it to the controller (instead of false/true)